app/core/templates/core/profile/description.html

```diff
@@ -71,6 +71,14 @@ <h2 class="mt0">{{ politician.first_name }} {{ politician.last_name }}</h2>
             </div>
         </div>
         {% endif %}
+        <div class="social-media-buttons">
+            <div class="facebook">
+                <a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.freedomvote.ch%2Fpolitician%2F{{politician.id}}%2F" target="_blank" title="{% trans "Share" %}" role="button" aria-label="{% trans "Share" %}" class="share-button">
```

Comment:
Is it possible to remove the hardcoded freedomvote.ch and replace it with the `BASE_URL`?

Reply:
@JohnL17 Yes please do that!

Reply:
@andreabettich Yes of course. Thank you.